Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -148,31 +148,38 @@ private void OnExecuteCommand(ExecuteCommandEvent evt)
             var selectedItem = m_ListView.GetRootElementForIndex(m_ListView.selectedIndex);
             if (selectedItem == null)
                 return;
-            switch (evt.commandName)
+
+            if (allowUICommandExecution)
             {
-                case CmdEvents.Rename:
-                    ((InputActionMapsTreeViewItem)selectedItem).FocusOnRenameTextField();
-                    break;
-                case CmdEvents.Delete:
-                case CmdEvents.SoftDelete:
-                    ((InputActionMapsTreeViewItem)selectedItem).DeleteItem();
-                    break;
-                case CmdEvents.Duplicate:
-                    ((InputActionMapsTreeViewItem)selectedItem).DuplicateItem();
-                    break;
-                case CmdEvents.Copy:
-                    CopyItems();
-                    break;
-                case CmdEvents.Cut:
-                    CutItems();
-                    break;
-                case CmdEvents.Paste:
-                    var isActionCopied = CopyPasteHelper.GetCopiedClipboardType() == typeof(InputAction);
-                    if (CopyPasteHelper.HasPastableClipboardData(typeof(InputActionMap)))
-                        PasteItems(isActionCopied);
-                    break;
-                default:
-                    return; // Skip StopPropagation if we didn't execute anything
+                switch (evt.commandName)
```

Comment:
Would recommend moving this switch statement into its own function for readability when there such a long switch statement